src/users.ts

```diff
@@ -50,6 +62,19 @@ export class Users {
     })
   }
 
+  /**
+   * Create a new Thread with name.
+   * @param name The name of the Thread.
+   * @param ctx Context containing gRPC headers and settings.
+   * These will be merged with any internal credentials.
+   */
+  async createThread(name: string, ctx?: Context) {
```

Comment:
Poking my nose in here :) 

I don't think I am understanding something here. Can't the "user" just leverage the threads client directly?

Reply:
Yes they can, this is just syntactic sugar for a common workflow

Reply:
Ok, but, two things: 
- You don't think it's confusing to have the same method in two places? 
- Looks like we're creating a DB here, so wouldn't it be `newDB`?

Reply:
I don't think it'll be confusing, because its possible they won't _ever_ actually touch the threads client at all if they're only ever using the user and buckets apis. This allows them to create a thread with less work (we had it the other way before and that's what has lead to some confusion it seems). To your second point... yup good point!

Reply:
I could certainly be convinced to revert this (as that was the original plan) but I'm trying to come up with _some_ way to make it so a developer who is "just trying to create a bucket for a user" doesn't have to think about databases etc.

Reply:
Hopefully that 'motivation' makes sense?

Reply:
At some point, that could defs make sense, yes. Right now, we're just trying to find a simple workflow that doesn't require too many steps to get started on the hub. I kind of 'like' being able to scope things out like this, so `import { Users }` to to do user things, `import { Buckets }` to do buckets things, and `import { Client }` to do threads things... with the ability to pass the same Context between all of these.

Reply:
Okay, sounds good. I think it might not be clear, or just plain confusing, what the role of the User API is. It likely only makes sense as a low level thing: The only thing you can't do with the threads and buckets clients is _list all threads created with your API key or Account session_ (since those clients are not multi-tenant by nature) which gave birth to this Users API thing. So, hiding it behind a higher level client feels smart... maybe there's just Buckets and Threads? Then, behind the scenes, the _two_ Users API methods are tacked onto Threads?

Reply:
So, two options: 
1. Alias one of the threads methods onto users (what you've done here)
2. Alias the two users methods onto threads (which would kill one of the app facing clients, which sounds like a good thing)

Reply:
I like it, let's do that (actually, now that you mention it, that was the original plan from @andrewxhill )

Reply:
Sweet, sounds good 🎸